Buildings/ThermalZones/Detailed/FLEXLAB/Rooms/Examples/TestBedX3WithRadiantFloor.mo

```diff
@@ -7,32 +7,80 @@ model TestBedX3WithRadiantFloor
   package Water = Buildings.Media.Water
     "Water model used in the radiant slab loop";
 
-  Buildings.ThermalZones.Detailed.FLEXLAB.Rooms.X3B.TestCell X3B(redeclare package Medium = Air,
+  Buildings.ThermalZones.Detailed.FLEXLAB.Rooms.X3B.TestCell X3B(redeclare
+      package                                                                      Medium = Air,
       nPorts=2,
-    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial) "Test cell X3B"
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    conExt(
+      each placeCapacityAtSurf_a=false,
```

Comment:
I set them to `false` to make sure we get the same behavior and the same reference results as on the master. I will set them to `true`. As a consequence the models will be changed to avoid over-determined system of equations. The reference results are expected to be different then.